src/automove.js

```diff
@@ -79,7 +79,10 @@ let getRepositioner = function( rule, cy ){
   } else if( r === 'drag' ){
     return dragAlong( rule );
   } else if( isObject( r ) ){
-    return boxPosition( r );
+    if(r['type'] == "inside")
```

Comment:
- Missing parentheses
- What about compatibility with the old API?
  - `{ x, y, type }` would be more compatible
  - Inside should be the default when no type is specified